jsonmerge_utils.py

```diff
@@ -51,14 +72,14 @@ def main():
     """
     try:
         configure_logger()
-        data_dir = input('Folder Path: ')
+        data_dir = args.filepath
         if not os.path.exists(data_dir):
             raise FolderNotFoundError
-        input_prefix = input('I/P Prefix: ')
+        input_prefix = args.ip
         if not os.path.isfile('{}{}'.format(path_creator(data_dir, input_prefix), '1.json')):
             raise FileNotFoundError
-        output_prefix = input('O/P Prefix: ')
-        max_file_size = int(input('Max File Size: '))
+        output_prefix = args.op
+        max_file_size = int(args.maxFileSize)

```

Comment:
both redundant, also as suggested above for args.maxFileSize, the use of _type_ parameter eliminates the need of  converting into int later on